### PR TITLE
Update qubes staging instructions

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
# Description

Tweak readme to allow for using client from source in sd-app

# Test Plan

- [x] changes look correct
- [ ] (optional) follow instructions to run from source in Qubes staging and verify that the client starts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,14 +201,15 @@ securedrop-client
 
 Or [manually initialize](https://github.com/freedomofpress/securedrop-client/blob/HEAD/files/securedrop-client) the SecureDrop Client database.
 
-8. To run a different version of the client, first add a NetVM (`sys-firewall`) to `sd-app` via its Qubes Settings so you can clone the client repository, and then follow these steps:
+8. To run a different version of the client, say the version from a branch called `<branchname>`, first add a NetVM (`sys-firewall`) to `sd-app` via its Qubes Settings so you can clone the client repository, and then follow these steps:
 
 ```
-git clone git@github.com:freedomofpress/securedrop-client.git
+sudo apt-get install git virtualenv
```
Comment (cfm) [inline README.md:207]: Nit: worth flagging that these won't persist in the appVM, although the working copy and virtual environment will you use them to create will?

```suggestion
sudo apt-get install git virtualenv  # NB. won't persist across "sd-app" reboots
```

**Review by cfm — APPROVED** [review]
Thanks for updating this, @zenmonkeykstop.  Looks good to me; left one nit inline for your consideration.

**Review by gonzalo-bulnes — DISMISSED** [review]
:shipit: (I second @cfm suggestion, as that's a command you'll have to run again when you restart `sd-app`. I think it's better to make sure people expect it so that no-one spends an hour wondering what's broken if they did all the setup the day before.)

**Comment by gonzalo-bulnes** [comment]
(applied the suggestion)

**Review by gonzalo-bulnes — APPROVED** [review]

**Pull request merged**